packages/aws-cdk-lib/aws-route53/lib/delete-existing-record-set-handler/index.ts

```diff
@@ -46,7 +46,7 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
     },
   });
 
-  await waitUntilResourceRecordSetsChanged({ client: route53, maxWaitTime: 60 }, { Id: changeResourceRecordSets?.ChangeInfo?.Id });
+  await waitUntilResourceRecordSetsChanged({ client: route53, maxWaitTime: 1800 }, { Id: changeResourceRecordSets?.ChangeInfo?.Id });
```

Comment:
Should be max 900, better yet 890 (to give Lambda 10s to send a response), and should match the defined `Timeout` of the Lambda function itself.